src/MOI_wrapper.jl

```diff
@@ -218,6 +218,63 @@ function MOI.optimize!(model::Optimizer)
 end
 
 
+const QP_SET_TYPES = (
+    MOI.GreaterThan{Float64},
+    MOI.LessThan{Float64},
+    MOI.EqualTo{Float64},
+    MOI.Interval{Float64}
+)
+
+const QP_FUNCTION_TYPES = (
+    MOI.SingleVariable,
+    MOI.ScalarAffineFunction{Float64}
+)
+
+const QP_OBJECTIVE_TYPES = (
+    MathOptInterface.ScalarAffineFunction{Float64},
+    MathOptInterface.ScalarQuadraticFunction{Float64},
+)
```

Comment:
```suggestion
const QP_SET_TYPES = Union{
    MOI.GreaterThan{Float64},
    MOI.LessThan{Float64},
    MOI.EqualTo{Float64},
    MOI.Interval{Float64},
}

const QP_FUNCTION_TYPES = {
    MOI.SingleVariable,
    MOI.ScalarAffineFunction{Float64},
}

const QP_OBJECTIVE_TYPES = {
    MathOptInterface.ScalarAffineFunction{Float64},
    MathOptInterface.ScalarQuadraticFunction{Float64},
}
```

To use subtype instead of tuple membership